### DISCUSSION TITLE
Welcome to tigramite Discussions!

### DISCUSSION THREAD

**Discussion by jakobrunge** [jakobrunge/tigramite#128] [Announcements]
## 👋 Welcome to Tigramite discussions!
  This is the place to raise questions and exchange on applications of tigramite methods. For bugs/feature requests please use the issues page. We hope that you:
  * Ask questions about how to apply tigramite for your type of problem setting.
  * Share ideas of what you already did with tigramite.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you did or intend to do with tigramite!

**Comment by jjakenichol** [comment]
Hey Jakob and the rest of the TIGRAMITE team, I'm a PhD student in computer science in New Mexico, USA. Since last January I've become very interested in causal discovery after having been frustrated with what machine learning can explain about a system. I am also an intern at Sandia National Laboratory. I began with some work applying random forests to understand Arctic sea ice and I'm working on a paper applying PCMCI to the same problem. Last month I started on a project to study the climate impacts of a volcanic eruption. My role is to see what we can learn with causal discovery, and my dissertation will be focused on how we can overcome the challenges that arise in applying causal discovery to such a large climate example. We'll be looking at SCMs and possibly the implementations in TETRAD, but PCMCI seems purpose built for our problem type so I'll be asking a lot of questions here.

Right now I'm reading Peters et al's book "Elements of Causal Inference Foundations and Learning Algorithms." It's a great book so far. Most causal inference books focus on more traditional causal inference, that's conducted in randomized/target trials and maybe briefly mention causal discovery. This book takes the opposite perspective, teaching causal discovery and relating it to traditional experimental design philosophies.
